formal/tla/lifecycle/counterexamples/perform-rewrite-tombstone-guard-probe.md

```
@@ -0,0 +1,67 @@
+whenever `listing.tombstone_key.is_some()`, refusing to rewrite the bucket at
+all. Round five's `PerformRewrite` guard (`~PresentObj("rwA") /\
+PresentObj("dreqR1") /\ ~PresentObj("doneR1") /\ superseded = {} /\ \A i \in
+Predecessors("rwA") : PresentObj(i)`) had no equivalent: `RetireBucket`
+(writing `tombB1`) has no dependency on `dreqR1`, `doneR1`, or `superseded`,
+so `tombB1` can be present while every one of those conjuncts still holds.
+
+## Probe module
+
+A scratch module under `/tmp`, `EXTENDS MCLifecycleGC`, adding:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)